recipe/meta.yaml

```
@@ -10,7 +10,7 @@ source:
   sha256: 7644e87afe9b94c32b4fca939d645c5b958d671691bd841d3391e31941090092
 
 build:
-  number: 0
+  number: 1
```

```suggestion
  number: 0
```

I don't think we need to bump the build number, as we already have set `{% set python_min = "3.10" %}`, and uses it in the script.